### PR TITLE
Updates + fixes for Neuronpedia

### LINKED ISSUE #4: [Bug] Multi-GPU / CUDA requires a tweak
I should create a PR for this but I don't want to lose track of this in the meantime:

When running on CUDA with multi-GPU, `SAEDashboard/sae_dashboard/feature_data_generator.py` needs a tweak:

Line 129
Before
```
model_acts = self.model.forward(minibatch_tokens, return_logits=False)
```

After
```
model_acts = self.model.forward(minibatch_tokens.to("cuda"), return_logits=False)
```

### PR DESCRIPTION
**Pull request by hijohnnylin**
Updates and Fixes:
	- make NeuronpediaDashboard objects
	- real acceptance test - test_simple_neuronpedia_runner()
		- runs gpt2-small res-jb and compares it with expected results
	- remove additional level of unnecessary batching
	- new settings
		- dtype override
		- shuffle tokens
	- minibatch device fix in feature_data_generator.py line 73. fixes #4 

You should be able to run `pytest -s tests/acceptance/test_neuronpedia_runner.py::test_simple_neuronpedia_runner` now which will check that the dashboard is correctly generated and that the output is (approximately) correct.

### CODE REVIEW

**Comment by jbloomAus** [comment]
Looks great!
